test/ens.js

```diff
@@ -358,7 +362,8 @@ contract("ENS contracts", (accounts) => {
     });
 
     it("should support registering ens for wallets created using the legacy wallet factory v1.6", async () => {
-      const factory = await WalletFactoryV16.new(registry.address, walletImplementation.address, ensManager.address);
+      const walletImpl = await BaseWalletV16.new();
+      const factory = await WalletFactoryV16.new(registry.address, walletImpl.address, ensManager.address);
```

Comment:
👍

Reply:
Also we need to restore the BaseWallet legacy tests but can be done in a separate PR.